drf_braces/serializers/form_serializer.py

```diff
@@ -41,9 +41,7 @@ def run_validation(self, data):
             # Only handle a ValidationError if the full validation is
             # requested or if field is in minimum required in the case
             # of partial validation.
-            if any([not self.parent.partial,
-                    self.parent.Meta.failure_mode == FormSerializerFailure.fail,
-                    self.field_name in self.parent.Meta.minimum_required]):
+            if self.field_name in self.parent.Meta.minimum_required:
```

Comment:
now it makes sense. you want to bypass the DRF field validations in order to force form validations to kick in. better place for that would be to add an option in the `Meta` which will force the serializer to not include any DRF fields and will force validation to happen in the form.

places to look in the code:

`FormSerializerBase .get_fields`
`FormSerializerOptions`